frontend/composables/useRoutingMap.ts

```diff
@@ -1,4 +1,6 @@
-import  maplibregl, { type LayerSpecification } from "maplibre-gl";
+
```

Comment:
Extra line here would be great to remove in a future commit :) And looks like the import statement's in the comment now :)